### PR TITLE
Fix small issues with README

### PR DESCRIPTION
**Pull request by echohack**
Signed-off-by: echohack <git@echohack.app>

Also note that this Helm chart will be announced in 1.13.0 as a "beta release", in line with our supported software policies: https://docs.pachyderm.com/latest/contributing/supported-releases/

I'm removing the "experimental" tag here as that's not the signal we want to send.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,46 +2,48 @@
...
- 2. Generate a Helm manifest with `helm template -f
-    examples/gcp-values.yaml ./pachyderm > helmmanifest.yaml`
- 3. Visually diff the two.
+1. Generate a `pachctl` manifest with
+
+```shell
+$ pachctl deploy googleBUCKET-NAME 10 --dynamic-etcd-nodes 1 -o yaml --dry-run > pachmanifest.yaml
+```
+
+1. Generate a Helm manifest with
```
Comment (robert-uhl) [inline README.md:23]: The number is resetting after each item. Is this perhaps related to indentation or whitespace?

**Review by robert-uhl — CHANGES_REQUESTED** [review]
Fix up the numbering and this looks good to me!

File: README.md (hunk 1 above)
Comment (echohack) [reply README.md:23]: This is correct. Numbered lists in markdown should always be listed as "1." and the markdown render will automatically generate the numbers.

Comment (echohack) [reply README.md:23]: In case you were more curious on how linters implement this:

https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md029---ordered-list-item-prefix

**Comment by chainlink** [comment]
Look good! I think we should ship after #40 (that tightens up most things enough that I'm comfortable removing 'experimental')

File: README.md (hunk 1 above)
Comment (robert-uhl) [reply README.md:23]: Agreed that numbered list numerals should all be 1 in the Markdown source; what I mean is that in GitHub they all _render_ as 1: take a look at https://github.com/pachyderm/helmchart/blob/a8ee4e8ca3df8cc97312f57c140a2eb5f3e73613/README.md

You need to indent the code blocks. For an example, see https://gist.github.com/robert-uhl/e0f0d866c338f8d89e5fb6b957f706da

Comment (echohack) [reply README.md:23]: Fixed!

**Comment by robert-uhl** [comment]
Looks great now! Thanks much.

**Comment by echohack** [comment]
DO-NOT-MERGE until #40 is merged.

**Comment by echohack** [comment]
Rebased. Should merge cleanly now.

**Review by robert-uhl — APPROVED** [review]
